### PR TITLE
LPS-174745 Filter list directly on the collection contributor

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by victorg1991** [comment]
ci:test:relevant

File: modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-inputs/src/main/java/com/liferay/fragment/collection/contributor/inputs/InputsFragmentCollectionContributor.java [hunk 1]
```diff
@@ -37,11 +44,41 @@ public String getFragmentCollectionKey() {
...
+
+		return ListUtil.filter(
+			fragmentEntries,
+			fragmentEntry ->
+				!Objects.equals(
+					fragmentEntry.getFragmentEntryKey(),
+					"INPUTS-rich-text-input") &&
+				!Objects.equals(
+					fragmentEntry.getFragmentEntryKey(), "INPUTS-textarea") &&
+				!Objects.equals(
```
Comment (ruben-pulido) [inline modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-inputs/src/main/java/com/liferay/fragment/collection/contributor/inputs/InputsFragmentCollectionContributor.java:77]: It should sorted alphabetically

**Comment by victorg1991** [comment]
ci:forward

**Pull request closed without merging**